aten/src/ATen/core/LegacyTypeDispatch.h

```diff
@@ -103,7 +103,6 @@ class CAFFE2_API LegacyTypeDispatch {
     type_registry[static_cast<int>(b)] = std::move(t);
     detail::getVariableHooks().registerVariableTypeFor(this, b);
   }
-private:
```

Comment:
Why'd this become public?

Reply:
For now, I'm calling this directly from Functions.h for all factory functions.